libr/debug/debug.c

```diff
@@ -668,6 +668,17 @@ R_API const char *r_debug_reason_to_string(int type) {
 	case R_DEBUG_REASON_FPU: return "fpu";
 	case R_DEBUG_REASON_STEP: return "step";
 	case R_DEBUG_REASON_USERSUSP: return "suspended-by-user";
+	case R_DEBUG_REASON_SIGALRM: return "alarm-signal";
```

Comment:
can you alphabetically sort all those cases?

Reply:
in the next PR